README.md

```diff
@@ -4,15 +4,22 @@ oVirt environment shutdown
 The `oVirt.shutdown-env` role iterates through all the entities (vms and hosts) in an oVirt/RHV cluster and performs a clean and ordered shutdown.
 It also handles an Hosted-Engine and hyper-converged GlusterFS environment as a special case automatically detecting it.
 The role is intended to be run only against the engine machine.
+Please note that host shutdown is async and the playbook terminates before HE hosts are really down.
 
 If on an Hosted-Engine environment, global maintenance mode will be set:
 the user has to manually exit it in order to get the engine VM automatically powered up once needed. 
 
 A startup mode is also available:
-in the startup mode the role will bring up all the IPMI configured hosts and it
+in the startup mode the role will bring up all the power management configured hosts and it
 will unset the global maintenance mode if on an hosted-engine environment.
 The startup mode will be executed only if the 'startup' tag is applied; shutdown mode is the default.
-The startup mode requires the engine to be already up.
+The startup mode requires the engine to be already up:
+power on it if it's a dedicated host, power on at least one of HE hosts (2 if on an hyperconverged env) and exit the global maintenance mode or manually start the engine VM with hosted-engine --vm-start
```

Comment:
change to capital 'P' -  change power to Power

Reply:
I also moved from . to : on line 16, not needed.